### ISSUE TITLE
Patreon

### ISSUE THREAD

**Issue by 0ryn** [ferdium/ferdium-app#1524]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that Ferdium Recipes adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22recipe+request+%3Asparkles%3A%22) for a recipe request that matches the one I want to file, without success.

### Name of the service

Patreon

### URL of the service

https://www.patreon.com

### Features you want implemented

Basically the same as the web page, maybe cut out the unnecessary view  in the app icons that pop up everywhere.
